src/lib/eldbus/eldbus_api.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _EFL_ELDBUS_API_H
+#define _EFL_ELDBUS_API_H
+
+#ifdef ELDBUS_API
+#error ELDBUS_API should not be already defined
+#endif
+
+#ifdef _WIN32
+# ifndef ELDBUS_STATIC
+#  ifdef ELDBUS_BUILD
+#   define ELDBUS_API __declspec(dllexport)
+#  else
+#   define ELDBUS_API __declspec(dllimport)
+#  endif
+# else
+#  define ELDBUS_API
+# endif
+# define ELDBUS_API_WEAK
+#else
+# ifdef __GNUC__
+#  if __GNUC__ >= 4
+#   define ELDBUS_API __attribute__ ((visibility("default")))
+#   define ELDBUS_API_WEAK __attribute__ ((weak))
```

Comment:
weak don't seems to be used.

Reply:
It seems that we are defining _weak_ for all libs and not necessarily using them

Reply:
yeah, I don't know if people at phab will like that.
Should we just do it like this for now, and later update it with phabs?

Reply:
Yeah I think. It will not be hard to remove definitions from all APIs if necessary